content/sensu-go/6.4/release-notes.md

```diff
@@ -85,6 +86,42 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.4.0 release notes
+
+**June 28, 2021** &mdash; The latest release of Sensu Go, version 6.4.0, is now available for download. 
+
+<!--// synopsis placeholder //-->
+
+See the [upgrade guide][1] to upgrade Sensu to version 6.4.0.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][215]) In the web UI, the system information modal now includes license expiration information, accessed via the `CTRL + .` keyboard shortcut, for users with the appropriate permissions.
+- ([Commercial feature][215]) Added [page-specific configuration][221] options and a custom [sign-in message attribute][220] for the web UI.
+- Added binary-only distribution for [macOS arm64][216].
+
+**IMPROVEMENTS:**
+
+- Added [etcd-log-level configuration flag][217] for setting the log level for the embedded etcd server.
+- Added [`wait` flag][218] for the `sensu-backend init` command, which indicates the backend should repeatedly try to establish a connection to etcd until it is successful.
+Also, the `timeout` flag for `sensu-backend init` is now treated as a duration instead of seconds, and values less than 1 second are converted to seconds.
```

Comment:
We should tell people what the duration format is (10s, 5m type values) and we should also tell them that if they put an integer value, it will be interpreted as seconds. (This isn't strictly true, if they put any value over 1 billion it will be interpreted as nanoseconds, but I think fibbing a bit to improve clarity will be useful)

Reply:
I need to add these details in the docs too

Reply:
@hillaryfraley do you have ideas for how to phrase this properly?